test/trento/application/integration/checks/checks_test.exs

```diff
@@ -287,4 +300,114 @@ defmodule Trento.Integration.ChecksTest do
 
     assert {:ok, catalog_by_provider} == Checks.get_catalog_grouped_by_provider()
   end
+
+  test "should handle execution started event properly" do
+    with_mock Trento.Commanded, dispatch: fn _, _ -> :ok end do
```

Comment:
we want to remove mock library soonish and leverage mox for this kind of things, could we add an issue about this test to keep track of the debt?

Reply:
At the end added simply a `#TODO` tag. Tech debt issue: https://github.com/trento-project/web/issues/146